docs/integrations/destinations/deepset.md

```diff
@@ -0,0 +1,40 @@
+# deepset
```

Comment:
Can you please fill out the docs so folks can understand what Deepset.ai is, why it's useful, and how to configure the destination?

Reply:
Done, thanks for your comments.